src/components/LGeoJson.vue

```diff
@@ -32,6 +38,17 @@ export default {
   beforeDestroy () {
     this.parentContainer.mapObject.removeLayer(this.mapObject);
   },
+  computed: {
+    mergedOptions () {
+      if (this.optionsStyle) {
+        const options = L.Util.extend({}, this.options)
```

Comment:
I think we should prefer ES6 syntax to do this, the whole block can be change with 
```
{
...this.options,
style: this.optionsStyle ? this.optionsStyle : this.options.style
}

Reply:
done! I use operator `||`.